bin/saya/src/persistent.rs

```diff
@@ -68,6 +74,29 @@ struct Start {
     /// Settlement network account private key
     #[clap(long, env)]
     settlement_account_private_key: Felt,
+
+    #[clap(long, env)]
+    db_dir: Option<PathBuf>,
+
+    #[clap(subcommand)]
+    pie_mode: PieGenerationMode,
+    #[clap(long, env, default_value_t = 60)]
```

Comment:
```suggestion
    pie_mode: PieGenerationMode,

    #[clap(long, env, default_value_t = 60)]
```
All with blank line, or none. To keep consistency.

Reply:
Would be great to add the doc comment (`///`) for each field to describe them and what they are for (with an `help` also when it applies to give more details from the CLI.